nav2_controller/src/nav2_controller.cpp

```diff
@@ -260,6 +260,9 @@ void ControllerServer::setPlannerPath(const nav_msgs::msg::Path & path)
 {
   RCLCPP_DEBUG(get_logger(),
     "Providing path to the controller %s", current_controller_);
+  if (path.poses.size() < 1) {
+    throw nav2_core::PlannerException("Invalid path, Path has length less than one.");
```

Comment:
I know its WIP, but just a quick comment: checking poses < 1, OK. But I think for the message it should be something a little less direct about the check like: "Path is empty" not that "path has length less than one".